app/views/StatusView.js

```diff
@@ -111,6 +111,7 @@ class StatusView extends React.Component {
 			const result = await RocketChat.setUserStatus(user.status, statusText);
 			if (result.success) {
 				logEvent(events.STATUS_CUSTOM);
+				store.dispatch(setUser({ statusText }));
```

Comment:
You should never touch `store` directly from components.
Add `setUser` to `mapDispatchToProps` and use it from `this.props.setUser` instead.
Same mistake on https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/views/StatusView.js#L168

Reply:
Done